eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Plutus/Evaluate.hs

```diff
@@ -350,9 +355,9 @@ evalTxExUnits ::
   --  sufficient execution budget.
   --  Otherwise, we return a 'TranslationError' manifesting from failed attempts
```

Comment:
This last part of the comment is outdated, since we don't return `Either`

Reply:
Same comment about the last line from haddock of the function, above: https://github.com/IntersectMBO/cardano-ledger/pull/4374/files#diff-d42005a7dd7c31638237a88fcf0ffe4feb1cca752f80b3d8901d8626e6d10f45L329